code/modules/economy/shippingmarket.dm

```diff
@@ -17,11 +17,25 @@
 	var/points_per_crate = 10
 
 	New()
+
+		add_commodity(new /datum/commodity/goldbar(src))
+
+		for(var/commodity_path in typesof(/datum/commodity))
```

Comment:
Why do gold bars need to be first? (That's what I'm inferring this stuff does, if it's something else please shout!)

If that is all that needs to happen, can do:
```
for (var/commodity_path in (typesof(/datum/commodity) - /datum/commodity/goldbar))
```

Reply:
Oh, good shout on the list subtraction

Goldbars need to be first because otherwise they sell as generic material bars instead of as gold bars